crates/sui-analytics-indexer/src/lib.rs

```diff
@@ -459,13 +478,17 @@ pub async fn make_move_call_processor(
     metrics: AnalyticsMetrics,
 ) -> Result<Processor> {
     let handler: Box<dyn AnalyticsHandler<MoveCallEntry>> = Box::new(MoveCallHandler::new());
-    let (epoch, starting_checkpoint_seq_num) =
-        read_store_for_epoch_and_checkpoint(config.remote_store_config.clone(), FileType::MoveCall)
-            .await?;
+    let starting_checkpoint_seq_num = if let Some(starting_checkpoint_seq_num) =
+        config.starting_checkpoint_seq_num
+    {
+        starting_checkpoint_seq_num
+    } else {
+        read_store_for_checkpoint(config.remote_store_config.clone(), FileType::Checkpoint).await?
+    };
```

Comment:
wonder if a function for this wouldn't be more clear

Reply:
makes sense